packages/sdk/src/oracle/README.md

```diff
@@ -99,14 +99,17 @@ export enum Flag {
   MissingRequest = "MissingRequest", // the client does not know the request, use client.setActiveRequest
   MissingUser = "MissingUser", // client does not have user data, use client.setUser
   WrongChain = "WrongChain", // user and request chain ids do not match, switch chains with client.switchOrAddChain
-  InProposeState = "InProposeState", // The on chain request is in a state where someone could propose, use client.proposePrice
-  InDisputeState = "InDisputeState", // The on chain request is in a stae where someone could dispute, use client.disputePrice
+  CanPropose = "CanPropose", // The on chain request is in a state where someone could propose, use client.proposePrice
+  CanDispute = "CanDispute", // The on chain request is in a state where someone could dispute, use client.disputePrice
+  CanSettle = "CanSettle", // The on chain request is in a stae where someone could settle the request.
```

Comment:
i renamed these to make more sense from the perspective of the user, since the state is actionable